### PR TITLE
README Add link to FDBP

### PR DESCRIPTION
**Pull request by davelab6**
:)

### CODE REVIEW

**Comment by davelab6** [comment]
cc @bobh0303

**Comment by chrissimpkins** [comment]
Thanks Dave and hello Bob!

Is it appropriate to add a direct link to the Versioning page of the FDBP site (https://silnrsi.github.io/FDBP/en-US/Versioning.html) rather than the home page?

**Comment by davelab6** [comment]
Yes :)

**Comment by chrissimpkins** [comment]
And these recommendations don't follow those best practices... I suppose the intent is to guide people who come across this repository to that Versioning information as the existing industry consensus on how to approach the versioning issue?

**Comment by bobh0303** [comment]
First, greetings and thanks for cc'ing me.

Second, yes you may certainly link directly to the Versioning page of the FDBP.

Finally, as to differences between FDBP and openfv, I was unaware of openfv effort until now. While I look forward to contributing as may be helpful, it will take me a bit to do a more in-depth review.

**Comment by davelab6** [comment]
Yes my intent was just to share a more informal introduction to
versioning... if the 2 approaches can be unified that would be fantastic :)

**Comment by chrissimpkins** [comment]
> if the 2 approaches can be unified that would be fantastic

Absolutely agree!  The intent here was definitely not to push a correct approach but to begin the discussion with an approach to versioning that deals with shortcomings and addresses needs that developers have based on our experience in the Hack project.  I would love to see this project evolve and mature, ideally becoming a Unified Versioning Scheme based upon involvement from a wide range of stakeholders.   Any/all interested are welcome to the organization account to work on it.

**Comment by chrissimpkins** [comment]
@bobh0303 Great to meet you Bob.  Really look forward to your feedback.  If you feel that this is an appropriate forum to use for the development of a unified versioning approach, I would be more than happy to send an invite to this organization account.  As noted above, this applies to any of the other authors of the SIL documentation and anyone else out there who cares about the topic and would like to use this organization account as a working group to address the lack of a comprehensive formal definition for how to version type.

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Software version strings commonly include the following semantics:
...
 
-These versioning concepts apply to the development of typeface software and are desirable during the development process; however, they are not all defined under the OpenType font versioning specification. Typeface software version strings are compiled into the nameID 5 record of the OpenType name table and the fontRevision record of the OpenType head table.  These records are defined as follows:
+These versioning concepts apply to the development of typeface software and are desirable during the development process; however, they are not all defined under the OpenType font versioning specification. Typeface software version strings are compiled into the nameID 5 record of the OpenType name table and the fontRevision record of the OpenType head table. These records are defined in the OpenType format specification as below, and explained in the SIL [Font Development Best Practices documentation](http://silnrsi.github.io/FDBP) ([source](https://github.com/silnrsi/FDBP).)
```
Comment (chrissimpkins) [inline README.md:32]:
```suggestion
These versioning concepts apply to the development of typeface software and are desirable during the development process; however, they are not all defined under the OpenType font versioning specification. Typeface software version strings are compiled into the nameID 5 record of the OpenType name table and the fontRevision record of the OpenType head table. These records are defined in the OpenType format specification as below, and explained in the SIL [Font Development Best Practices documentation](https://silnrsi.github.io/FDBP/en-US/Versioning.html) ([source](https://silnrsi.github.io/FDBP/en-US/Versioning.html).)
```

Comment (chrissimpkins) [reply README.md:32]: modified the link to the Versioning documentation as discussed in this thread

**Review by chrissimpkins — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chrissimpkins) [inline README.md:32]:
```suggestion
These versioning concepts apply to the development of typeface software and are desirable during the development process; however, they are not all defined under the OpenType font versioning specification. Typeface software version strings are compiled into the nameID 5 record of the OpenType name table and the fontRevision record of the OpenType head table. These records are defined in the OpenType format specification as below, and explained in the SIL [Font Development Best Practices documentation](https://silnrsi.github.io/FDBP/en-US/Versioning.html) ([source](https://github.com/silnrsi/FDBP).)
```

Comment (chrissimpkins) [reply README.md:32]: reverted change to the link to the source...

**Review by chrissimpkins — APPROVED** [review]

**Comment by davelab6** [comment]
Great changes!

**Pull request merged**